src/lxc/lxc-start-ephemeral.in

```diff
@@ -36,6 +36,10 @@ import tempfile
 _ = gettext.gettext
 gettext.textdomain("lxc-start-ephemeral")
 
+def printstderr(*args):
+    print("lxc-start-ephemeral is deprecated in favor of lxc-copy\n", *args, file = sys.stderr)
```

Comment:
Pretty sure pep8 would be unhappy with that "file = sys.stderr", should be "file=sys.stderr".

For python changes, it's usually a good idea to make sure that the code is clean when running "pep8" and "pyflakes3" on it.